src/webui/src/App.tsx

```diff
@@ -66,13 +67,12 @@ class App extends React.Component<{}, AppState> {
             }
         }
     }
-    
+
     changeInterval = (interval: number): void => {
         this.setState({ interval });
-        if (this.timerId === null && interval !== 0) {
-            window.setTimeout(this.refresh);
-        } else if (this.timerId !== null && interval === 0) {
-            window.clearTimeout(this.timerId);
+        if (this.flag === false) {
+            this.refresh();
```

Comment:
`refresh()` will setup a (sequence of) timer. If I spam changing interval, there will be bunch of timers.

Reply:
had updated.